src/api/app/components/bs_request_overview_avatars_component.html.haml

```diff
@@ -0,0 +1,12 @@
+%ul.list-inline.d-flex.flex-row-reverse.avatars.m-0
+  - avatar_objects.each do |avatar_object|
+    %li.list-inline-item
+      = helpers.image_tag_for(avatar_object, size: 23, custom_class: 'rounded-circle bg-light border border-gray-400')
+
+  - if avatar_objects.empty? && @review.for_package?
+    %li.list-inline-item
+      %span.fa.fa-archive.text-warning.rounded-circle.bg-light.border.border-gray-400.simulated-avatar
+  - elsif avatar_objects.empty? && @review.for_project?
+    %li.list-inline-item
+      %i.fa.fa-cubes.text-secondary
```

Comment:
This line should be removed to avoid having duplicated icons:

![duplicated-icon](https://user-images.githubusercontent.com/1102934/184104539-eb5dffe8-227e-47df-8e15-89f1ee33d1fb.png)

Reply:
It's updated